.github/workflows/project-trigger.yml

```
@@ -0,0 +1,30 @@
+name: Reconciles Project Versions and triggers them
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
```

```suggestion
    - cron: '0 0 * * *'
```

We changed design to run daily rather than weekly.